README.md

```diff
@@ -24,6 +23,19 @@ There are a few things that don't currently work in `redis-oplog` when using the
 - Custom namespaces and channels ([`redis-oplog` issue #279](https://github.com/cult-of-coders/redis-oplog/issues/279))
 - Synthetic mutations ([`redis-oplog` issue #277](https://github.com/cult-of-coders/redis-oplog/issues/277))
 
+## Nix
+
+We are using Nix at Tulip so we're rolling a `default.nix` and a `flake.nix` wrapper around it for Flake support. The `default.nix` file contains a `vendorHash` attribute, which would be the calculated hash of the downloaded modules to be vendored. To update this value after a `go.sum` file change (which means that different modules were downloaded), refer to the following snippet:
+
+```bash
+# TODO: default.nix: Change the `vendorHash` property to an empty string and run `nix build`.
+nix build
+# ...
+# error: hash mismatch in fixed-output derivation '/nix/store/by8bc99ywfw6j1i6zjxcwdmc5b3mmgzv-oplogtoredis-3.0.0-go-modules.drv':
```

Comment:
????

Reply:
Yeah, try reading the updated instructions, this was highly cryptic. I hope that the new one is more understandable ^^